interfaces/builtin/dsp.go

```diff
@@ -62,6 +68,7 @@ const ambarellaDspConnectedPlugApparmor = `
 
 var ambarellaDspConnectedPlugUDev = []string{
 	`KERNEL=="iav"`,
+	`KERNEL=="cavalry`,
```

Comment:
before we merge this can you confirm the output of `udevadm info --query property /dev/cavalry` so we can confirm that this is the right rule to use? It probably is, I've never heard of anything else having a cavalry device node, but just to confirm.

Reply:
@anonymouse64 definitely agree, actually I did captured subsystem before I created this PR:

```
$ udevadm info --query property /dev/cavalry
DEVPATH=/devices/virtual/cavalry/cavalry
DEVNAME=/dev/cavalry
DEVMODE=0666
MAJOR=245
MINOR=0
SUBSYSTEM=cavalry
```

Reply:
thanks, the rule you provided for `KERNEL=="cavalry"` is fine then, thanks for confirming this